### PR TITLE
Migration from Ostrich to Metrics

### PR DESCRIPTION
**Pull request by AnishShah**
@szymonm please have a look.

### CODE REVIEW

**Comment by AnishShah** [comment]
@szymonm 
`"com.twitter" % "twitter-server_2.10" % "1.9.0"` gives ```java.lang.RuntimeException: Conflicting cross-version suffixes in: com.twitter:util-hashing, com.twitter:finagle-core, com.twitter:finagle-http, com.twitter:util-codec, com.twitter:util-jvm, com.twitter:util-collection, com.twitter:util-core, com.twitter:util-logging, com.twitter:util-app```

Whereas 

`"com.twitter" %% "twitter-server" % "1.9.0"` gives no twitter-server found

**Comment by szymonm** [comment]
Hi @AnishShah the problem is that last published version of twitter-server for 2.9 is 1.7.3. http://search.maven.org/#search%7Cga%7C1%7Ctwitter-server
You are trying to use 1.9.0.

You should either use 1.7.3. version or migrate everything to 2.10 and change ostrich to twitter-server in a single PR.
What do you think @pankajgupta ?

**Comment by AnishShah** [comment]
@szymonm I tried migrating everything to 2.10 and it is working on local machine. But Travis fails. Any help ? Am I doing anything wrong?

**Comment by szymonm** [comment]
Can you show Travis logs?

**Comment by szymonm** [comment]
Did you try using 1.7.3?

File: project/Build.scala [hunk 1]
```diff
@@ -142,16 +123,9 @@ object Cassovary extends Build {
...
-          case x if x startsWith "2.10" => "2.10"
-          case x => x
-        },
-        "com.twitter" %% "finagle-ostrich4" % "6.20.0"  cross CrossVersion.binaryMapped {
-          case "2.9.3" => "2.9.2"
-          case x if x startsWith "2.10" => "2.10"
-          case x => x
-        }
+        "com.twitter" %% "finagle-http" % "6.20.0",
+        "com.twitter" %% "finagle-ostrich4" % "6.20.0",
```
Comment (szymonm) [inline project/Build.scala]: you should be able to delete it, don't you?

**Comment by AnishShah** [comment]
```
sbt.ResolveException: unresolved dependency: com.twitter#util-core_2.9.3;6.16.0: not found
unresolved dependency: com.twitter#ostrich_2.9.3;9.1.0: not found
unresolved dependency: com.twitter#util-logging_2.9.3;6.12.1: not found
sbt.ResolveException: unresolved dependency: com.twitter#util-core_2.9.3;6.16.0: not found
unresolved dependency: com.twitter#ostrich_2.9.3;9.1.0: not found
unresolved dependency: com.twitter#util-logging_2.9.3;6.12.1: not found
```
I think, Travis fails because it is running `sbt ++2.9.3 test` command.

**Comment by AnishShah** [comment]
Using 1.9.0 instead of 1.7.3 would be better, right?

**Comment by szymonm** [comment]
Change line in .travis.yml

**Comment by szymonm** [comment]
Try to unify twitter utils version, ex: ulti-core, util-logging, util-app, finagle-http (if possible)

**Comment by szymonm** [comment]
I thought you don't want to migrate to 2.10 yet, thus I suggested twitter-sever 1.7.3

**Comment by caniszczyk** [comment]
I would just move to 2.11 if that's now possible, older dependencies will be hard to find on ~2.9

**Comment by szymonm** [comment]
@AnishShah looking at https://github.com/twitter/finagle/blob/master/project/Build.scala may help you.

**Comment by pankajgupta** [comment]
Yes, feel free to remove scala 2.9 totally in a separate PR.

**Comment by AnishShah** [comment]
@pankajgupta OK. I will do that first then.

**Comment by AnishShah** [comment]
Closing this.. I'll send a new PR ..

**Pull request closed without merging**